README.md

```diff
@@ -148,207 +171,247 @@ Klaviyo.createEvent(event)
 
 ## Push Notifications
 
-### Prerequisites:
-
-- Firebase account
-- Familiarity with [Firebase](https://firebase.google.com/docs/cloud-messaging/android/client)
-  documentation.
+### Prerequisites
+- A [Firebase account](https://firebase.google.com/docs/android/setup) for your Android app.
+- Familiarity with [Firebase](https://firebase.google.com/docs/cloud-messaging/android/client) documentation.
```

Comment:
When I was integrating I wasn't sure what part of this doc I need to familiarize myself with. This probably makes sense to an android dev but does it make sense to elaborate a little here?

Reply:
At the end of the day, our push SDK is a wrapper around the FCM SDK. So I think anyone integrating our SDK should have a full understanding of FCM first. 
If someone sends us an issue that is clearly explained in there, I'd like to be able to justifiably say "read the official FCM documentation that we linked in the readme"